mm/ktsan/ktsan.h

```diff
@@ -199,10 +201,14 @@ struct kt_thr_s {
 	kt_trace_t		trace;
 	int			call_depth;
 	struct list_head	quarantine_list;
+	int			event_depth;
```

Comment:
Please give it more descriptive name. event_depth is a wrong name for what it does. It could be something along the lines of "disable_depth".